Libraries/Lists/FlatList.js

```diff
@@ -97,39 +118,43 @@ type OptionalProps<ItemT> = {|
     ...
   },
   /**
-   * If true, renders items next to each other horizontally instead of stacked vertically.
+    If true, renders items next to each other horizontally instead of stacked vertically.
    */
   horizontal?: ?boolean,
   /**
-   * How many items to render in the initial batch. This should be enough to fill the screen but not
-   * much more. Note these items will never be unmounted as part of the windowed rendering in order
-   * to improve perceived performance of scroll-to-top actions.
+    How many items to render in the initial batch. This should be enough to fill the screen but not
+    much more. Note these items will never be unmounted as part of the windowed rendering in order
+    to improve perceived performance of scroll-to-top actions.
    */
   initialNumToRender: number,
   /**
-   * Instead of starting at the top with the first item, start at `initialScrollIndex`. This
-   * disables the "scroll to top" optimization that keeps the first `initialNumToRender` items
-   * always rendered and immediately renders the items starting at this initial index. Requires
-   * `getItemLayout` to be implemented.
+    Instead of starting at the top with the first item, start at `initialScrollIndex`. This
+    disables the "scroll to top" optimization that keeps the first `initialNumToRender` items
+    always rendered and immediately renders the items starting at this initial index. Requires
+    `getItemLayout` to be implemented.
    */
   initialScrollIndex?: ?number,
   /**
-   * Reverses the direction of scroll. Uses scale transforms of -1.
+    Reverses the direction of scroll. Uses scale transforms of `-1`.
    */
   inverted?: ?boolean,
   /**
-   * Used to extract a unique key for a given item at the specified index. Key is used for caching
-   * and as the react key to track item re-ordering. The default extractor checks `item.key`, then
-   * falls back to using the index, like React does.
+    ```jsx
+    (item: object, index: number) => string;
+    ```
+
+    Used to extract a unique key for a given item at the specified index. Key is used for caching
+    and as the react key to track item re-ordering. The default extractor checks `item.key`,
+    then falls back to using the index, like React does.
    */
   keyExtractor: (item: ItemT, index: number) => string,
   /**
-   * Multiple columns can only be rendered with `horizontal={false}` and will zig-zag like a
-   * `flexWrap` layout. Items should all be the same height - masonry layouts are not supported.
+    Multiple columns can only be rendered with `horizontal={false}` and will zig-zag like a `flexWrap` layout.
+    Items should all be the same height - masonry layouts are not supported.
    */
   numColumns: number,
   /**
-   * See `ScrollView` for flow type and further documentation.
+   See `ScrollView` for flow type and further documentation.
```

Comment:
This prop(fadingEdgeLength) is not present in the markdown file. I wonder if this has to be synced with the markdown later or has to be removed from here(considering the docs to be source of truth).

Reply:
@ayushjainrksh Looks like the `fadingEdgeLength` property is [still being used](https://github.com/facebook/react-native/blob/03489539146556ec5ba6ba07ac338ce200f5b0f4/ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactHorizontalScrollViewManager.java#L294-L302), so we should have it documented. 

Ideally, what we should do is figure out the actual behaviour of the prop and document it here. Based on the implementation code, it calls to [Android.View.setFadingEdgeLength](https://developer.android.com/reference/android/view/View#setFadingEdgeLength(int)), e.g. something like:

> Set the size of the faded edge used to indicate that more content in this view is available. 

Congratulations, we've found a first case of missing documentation on the website, so this update will make the docs a bit better!

Reply:
@jevakallio I looked into the behaviour of fadingEdgeLength and found the [PR#26163](https://github.com/facebook/react-native/pull/26163/) in which the prop was added. As far as I understand, this prop provides same functionality to ScrollView and FlatList. So I would suggest something like:

```
/**
   Fades out the edges of the the list content.

   If the value is greater than 0, the fading edges will be set accordingly to the current scroll 
   direction and position, indicating if there is more content to show.

   @platform android
*/
```
I tried to implement this in snackplayer but it didn't work. So I am not sure about the functionality.

Reply:
The documentation you've written looks great, but I am also not able to get it to work on FlatList or ScrollView. My test Snack is here: https://snack.expo.io/@jevakallio/trusting-bagel

Would have to investigate why this doesn't work, but for now I say we do include the documentation as you've proposed and log a new issue on the main RN repository with the above Snack link as a reproduction.

@ayushjainrksh do you want to open the issue or shall I?

Reply:
@jevakallio Thanks, I tried to implement it but couldn't make it work either. I can open the issue. By main RN repo do you mean facebook repo?

Reply:
@ayushjainrksh correct, since this bug is outside of the MLH scope